src/app/zap-templates/templates/chip/helper.js

```diff
@@ -483,6 +483,8 @@ function asObjectiveCNumberType(label, type)
       default:
         error = label + ': Unhandled underlying type ' + zclType + ' for original type ' + type;
         throw error;
+        case 'fabricId':
+        case 'chip::FabricId':
```

Comment:
I don't follow this change.  Apart from the weird indent, it seems like it would return nothing from a function that expects a string return value...

Reply:
This was accidentally leftover when I was deving. Removing